idprovider/local/idprovider_test.go

```diff
@@ -254,9 +254,9 @@ func Test_YAML_UpdateStorage(t *testing.T) {
 	})
 
 	t.Run("happy_non_empty_file", func(t *testing.T) {
-		fileWithTwoPeers := contactstest.NewYAMLFileT(t, peer1, peer2)
-		fileWithThreePeers := contactstest.NewYAMLFileT(t, peer1, peer2, peer3)
-		c, err := contactsyaml.New(fileWithTwoPeers, walletBackend)
+		fileWithTwoPeers := idprovidertest.NewIDProviderT(t, peer1, peer2)
```

Comment:
```suggestion
		fileWithTwoPeerIDs := idprovidertest.NewIDProviderT(t, peer1, peer2)
```


Using peer ID instead of peer. Other usage of the variable also needs to be updated.

Reply:
Updated in all places